scripts/imgtool.py

```diff
@@ -122,7 +122,8 @@ def args():
     getpub.add_argument('-k', '--key', metavar='filename', required=True)
     getpub.add_argument('-l', '--lang', metavar='lang', default='c')
 
-    sign = subs.add_parser('sign', help='Sign an image with a private key')
+    sign = subs.add_parser('sign', help='Sign an image with a private key',
+            aliases=['create'])
```

Comment:
I think a user would need some more documentation to understand this.

The help text for 'create' would now state that this would sign an image, wouldn't it?

Reply:
I agree that just a little more help text would be nice.

Reply:
I have the same opinion.